core/src/main/java/org/pentaho/platform/dataaccess/datasource/wizard/public/datasourceAdminDialog.xul

```diff
@@ -22,7 +22,8 @@
 
       <spacer flex="1"/>
 
-      <button id="datasourceDropdownButton" image="images/spacer.gif" pen:classname="pentaho-optionsdatasourcebutton">
+      <button id="datasourceDropdownButton" image="images/spacer.gif" pen:classname="pentaho-optionsdatasourcebutton"
```

Comment:
for all new properties, found image in application and used engineering discretion for acceptable text